source/_integrations/switcher_kis.markdown

```diff
@@ -62,3 +62,8 @@ Note: This does not affect the auto off timer.
 | ------------- | --------- | -------------------------------------------------------------------------------------- | -------------------------- |
 | `entity_id`   | Yes       | Name of the entity id associated with the integration, used for permission validation  | switch.switcher_kis_boiler |
 | `timer_minutes`    | Yes       | Integer containing timer minutes (valid range 1 to 150)                                      | 90                    |
+
+
+## Notes
+
+If Home Assistant and your Switcher are not on the same network, you need to create a firewall rule, which allows a connection on port 20002 with the UDP protocol from Switcher to your TV.
```

Comment:
```suggestion
If Home Assistant and your Switcher are not on the same network, you need to create a firewall rule, which allows a connection on port 20002 with the UDP protocol from Switcher to your Home Assistant and port 9957 with the TCP protocol from Home Assistant to your Switcher.
```

Reply:
Switcher doesn't really have any configurations to set a firewall.  Its an esp essentially.  I didn't need to set such a setting on the homesssistant firewall

Reply:
Your installation does not have a firewall for outgoing connections, but some users run devices on a separate IOT VLAN and needs to create firewall rules also for outgoing connections not only for incoming connections.
If we add documentation for connections we need to document all the ports needed. It is up to the user to verify what he needs in his installation.

Reply:
Ok, understood. merged